R-proj/man/vol_R.Rd

```diff
@@ -0,0 +1,78 @@
+% Generated by hand
```

Comment:
why not from roxygen?

Reply:
Because RcppExports.R is auto generated in every build(). So roxygen comments are destroyed in every build().